src/components/WhatWeDo/WhatWeDo.js

```diff
@@ -0,0 +1,15 @@
+import React, { Component } from 'react';
+import '../../styles/What.css';
+import Blurbs from "./Blurbs.js"
+
+class WhatWeDo extends Component {
+  render() {
+    return (
+      <div className="WhatWeDo" id="WhatWeDo">
+        <Blurbs />
+      </div>
+    );
+  }
+}
+
+export default WhatWeDo;
```

Comment:
WhatWeDo.js: Code is clean and well-organized for future additions/amendments. Please do the same for OurStory.js by making updates to About.js and creating OurStory.js.